tests/yaml_parse_file_002.phpt

```diff
@@ -22,6 +22,6 @@ Warning: yaml_parse_file(): Filename cannot be empty in %s on line %d
 
 Warning: yaml_parse_file(): Filename cannot be empty in %s on line %d
 
-Warning: yaml_parse_file(invalid): failed to open stream: No such file or directory in %s on line %d
+Warning: yaml_parse_file(invalid): %sailed to open stream: No such file or directory in %s on line %d
```

Comment:
A possible alternative could be checking for an arbitrary single character (`%c`), or for a regex character class (`%r[Ff]%r`).

Reply:
@cmb69 Thanks, I didn't know about `%r` :)

Reply:
All supported tags are documented at http://qa.php.net/phpt_details.php#expectf_section :)